libs/s25main/desktops/dskCredits.cpp

```diff
@@ -82,69 +89,178 @@ dskCredits::dskCredits() : Desktop(LOADER.GetImageN("setup013", 0))
+      "Oliver Lang",                    // 
+      "Marius Loewe",                   // 
+      "Eric Lutter",                    // 
+      // M                              // 
+      "Jan Montag",                     // 
+      "Kai Müller",                     // 
+      "morlock",                        // 
+      "Jan Mozgawa",                    // 
+      // N                              // 
+      // O                              // 
+      // P                              // 
+      "Wojciech Pieklik",               // 
+      "Mike Plackowski",                // 
+      "Daniel Platt",                   // 
+      // Q                              // 
+      // R                              // 
+      "Philip Rau",                     // 
+      "Ronny Richter",                  // 
+      // S                              // 
+      "Daniel Seindl",                  // 
+      "Vasilyev Sergey",                // 
+      "Patrick Schefczyk",              // 
+      "Marcel Schneider",               // 
+      "Alexander Schoedon",             // 
+      "Max Skuratov",                   // 
+      "Philipp Strathausen",            // 
+      "Benjamin Stoisch",               // 
+      "Felix Stolle",                   // 
+      // T                              // 
+      "Nina Tillmann",                  // 
+      "Angelo Tiegs",                   // 
+      // U                              // 
+      // V                              // 
+      // W                              // 
+      "Niels Wiederanders",             // 
+      "Philipp Wohlleben"               // 
+      // X                              // 
+      // Y                              // 
+      // Z                              // 
+    };
+
+    // alphabetical again, two rows
     entry = CreditsEntry(_("Donators"), _("Thank you for your donations!"));
-    entry.lines.push_back(_("various anonymous donators"));
-    entry.lines.push_back("Markus Becker");
-    entry.lines.push_back("Karsten Backhaus (K-Duke)");
-    entry.lines.push_back("Patrick Haak (Demophobie)");
-    entry.lines.push_back("Gilles Bordelais");
-    entry.lines.push_back("Dominic Jonas");
-    entry.lines.push_back("Rene Hopf");
-    entry.lines.push_back("Christopher Kuehnel (Spikeone)");
-    entry.lines.push_back("Philipp Strathausen");
-
-    entry.lines.push_back(CreditsEntry::Line("Max Skuratov", 1));
-    entry.lines.push_back(CreditsEntry::Line("Marius Loewe", 1));
-    entry.lines.push_back(CreditsEntry::Line("Eric Lutter", 1));
-    entry.lines.push_back(CreditsEntry::Line("Bob Kromonos Achten", 1));
-    entry.lines.push_back(CreditsEntry::Line("morlock", 1));
-    entry.lines.push_back(CreditsEntry::Line("Hans Gabathuler", 1));
-    entry.lines.push_back(CreditsEntry::Line("Jan Montag", 1));
-    entry.lines.push_back(CreditsEntry::Line("Patrick Schefczyk", 1));
-    entries.push_back(entry);
+
+    const size_t page_size = 15;
```

Comment:
You use 2 columns but 15 names? Why not 16 so it is 8 per column?

Also: Maybe alternate the names (1 left, 1 right, 1 left, ...) so the last page is filled top to bottom. But doesn't really matter

Reply:
because its not filled that way.
we have on the left 2 additional lines with the red "bottom" text - and below that the bobs will start walking around.
this way we have a quite good layout with the names and the red text.